cmake/modules/CheckCompiler.cmake

```diff
@@ -206,6 +206,15 @@ if(gnuinstall)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DR__HAVE_CONFIG")
 endif()
 
+# Enable ThinLTO for clang in release build modes if the current clang version
+# supports it. We don't add it to Release mode because it increases build times.
```

Comment:
Did you mean *"We don't add it to the __Debug__ mode because it increases build time"*?